MAUI/Circular-Charts/GroupTo.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: Grouping data points in .NET MAUI Chart control | Syncfusion
+description: This section explains about how to group data points in Syncfusion .NET MAUI Chart (SfCircularChart) control.
+platform: maui
+control: SfCircularChart
+documentation: ug
+---
+
+# Grouping Data Points in .NET MAUI SfCircularChart
+
+The small segments in the circular chart can be grouped into an `Others` category using the [GroupTo]() and [GroupMode]() properties of the circular chart. The [GroupMode]() property is used to specify the grouping type based on slice angle, actual data point value, or percentage. The [GroupTo]() property is used to set the limit to group data points into a single slice. The grouped segment is labeled as `Others` in the chart legend.
+
```

Comment:
Mention notes that group to only support for pie and doughnut.

Reply:
Notes added